### ISSUE TITLE
[BUG] Cannot print certain characters in 0.4.2

### ISSUE THREAD

**Issue by corndog2000** [404background/node-red-contrib-python-venv#37]
I used to be able to print arbitrary characters from a venv node. But, after updating to 0.4.2 I cannot print the Ω character anymore. I get a python crash.

Example flow:
```json
[{"id":"ca4269441efb4d4c","type":"tab","label":"Test","disabled":false,"info":"","env":[]},{"id":"79648dec4052841a","type":"inject","z":"ca4269441efb4d4c","name":"","props":[],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","x":200,"y":180,"wires":[["1c957cd89fead013"]]},{"id":"1c957cd89fead013","type":"venv","z":"ca4269441efb4d4c","venvconfig":"a7181e9a34e8625e","name":"","code":"sheet_resistance = 100\n\nprint(f'GAMMA Sheet resistance: {sheet_resistance} Ω/square')","continuous":false,"x":350,"y":180,"wires":[["9e31cf1f56adbed4"]]},{"id":"9e31cf1f56adbed4","type":"debug","z":"ca4269441efb4d4c","name":"debug 15","active":true,"tosidebar":true,"console":true,"tostatus":false,"complete":"payload","targetType":"msg","statusVal":"","statusType":"auto","x":520,"y":180,"wires":[]},{"id":"a7181e9a34e8625e","type":"venv-config","venvname":"spectrometers","version":"default"}]
```

Error message:

```python
Error 1: Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<string>", line 3, in <module>
  File "C:\Python312\Lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeEncodeError: 'charmap' codec can't encode character '\u03a9' in position 28: character maps to <undefined>
```

System information:

- Windows 11 x64 23H2
- Node-RED version: v4.0.3
- Node.js  version: v20.17.0
- node-red-contrib-python-venv version: 0.4.2
- Python version: 3.12.6 (tags/v3.12.6:a4a2d2b, Sep  6 2024, 20:11:23) [MSC v.1940 64 bit (AMD64)]

**Comment by corndog2000** [comment]
Error message:

Error 1: Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<string>", line 3, in <module>
  File "C:\Python312\Lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeEncodeError: 'charmap' codec can't encode character '\u03a9' in position 28: character maps to <undefined>

**Comment by corndog2000** [comment]
System information:

- Windows 11 x64 23H2
- Node-RED version: v4.0.3
- Node.js  version: v20.17.0
- node-red-contrib-python-venv version: 0.4.2
- Python version: 3.12.6 (tags/v3.12.6:a4a2d2b, Sep  6 2024, 20:11:23) [MSC v.1940 64 bit (AMD64)]

**Comment by 404background** [comment]
Thank you @corndog2000  for your report to the issue!

Thanks also for the information on the sample flow and system.
The reason why you cannot run it after changing the version may have something to do with #26 .

I ran that flow on my Windows 10/11 and Python 3.8/3.10 and did not get the same error.
However, when I changed the ohm character to an environment dependent character and ran it, I got the following error.

``UnicodeEncodeError: 'cp932' codec can't encode character '\u2126' in position 28: illegal multibyte sequence``
![image](https://github.com/user-attachments/assets/34df02ec-ecd0-4dcc-892d-2cb71ed75fe1)

I will address this error.

**Comment by 404background** [comment]
@corndog2000 
In #38 , I fixed the UnicodeEncodeError.
It was also displayed without error for Ω.
![スクリーンショット 2024-11-02 230442](https://github.com/user-attachments/assets/e444569a-86fb-4330-84fc-67b9679ef52f)

I have released it as 0.5.0 and would appreciate your checking it out.
Please feel free to contact me if the error appears again!
